homeassistant/components/media_player/directv.py

```diff
@@ -104,28 +136,43 @@ def __init__(self, name, host, port, device):
         self._last_position = None
         self._is_recorded = None
         self._assumed_state = None
+        self._available = False
```

Comment:
@MartinHjelmare, no specific reason. I can move it into another PR. My only thought here was cause it is a small change to just add it here. Based on my understanding otherwise I need to wait till this PR is merged before then submitting the next one right?

Reply:
You can base one on the other and submit both, but put one as WIP, and then rebase that one when the other is merged.

Or wait, like you say, and submit the second PR when the first is merged.